packages/teams-js/src/public/sharing.ts

```diff
@@ -194,4 +194,80 @@ export namespace sharing {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.sharing ? true : false;
   }
+
+  /**
+   * Namespace to get the list of content shared in a context
+   *
+   * @beta
+   */
+  export namespace history {
+    /**
+     * Represents IContentResponse parameters.
+     *
+     * @beta
+     */
+    export interface IContentResponse {
+      /** Id of the app where the content was shared from */
+      appId: string;
+      /** Title of the shared content */
+      title: string;
+      /** Reference of the shared content */
+      contentReference: string;
+      /** Id of the thread where the content was shared */
+      threadId: string;
+      /** Id of the user who shared the content */
+      author: string;
+      /** Type of the shared content */
+      contentType: string;
```

Comment:
Can this really be any string? Or is there a list of possible content types? Is it a MIME type?

Reply:
These are string types. Right now on client the contentType is "ShareToStage" for this scenario.

Reply:
So there's only one possible value? Can you document the valid values so that developers know what to expect?

Reply:
Added validation and updated documentation.